ajax/libs/dayjs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "dayjs",
   "filename": "dayjs.min.js",
-  "version": "1.5.24",
```

Comment:
According to https://github.com/cdnjs/cdnjs/pull/12835#pullrequestreview-129262487,
You can leave `version` unmodified and don't need to submit `1.6.9/dayjs.min.js` as a result.
(`version` and `filename` changes are followed by submitted files.)